src/components/adopters.tsx

```diff
@@ -70,6 +71,11 @@ const adopters = [
     image: spiribo,
     url: 'https://www.spiri.bo/',
   },
+  {
+    title: 'Data Detect',
+    image: datadetect,
+    url: 'http://datadetect.com/',
```

Comment:
Then use e.g. 
```suggestion
    url: 'https://unifiedglobalarchiving.com/data-detect/',
```
but we really can't have non https links on our website.

Reply:
ok I will talk to orly if that is good too. 
he told me to use the datadetect.com cause they might switch to a dedicated website for it

Reply:
ok but it won't help them now if we have the http URL there, they should add a redirect over https as well